### ISSUE TITLE
Error when extending user store with getters.

### ISSUE THREAD

**Issue by jimcreate78** [DivanteLtd/vue-storefront#1028]
When I tried to extend the user store adding getters, I would get a vuex error that isLoggedIn was defined more than once.  Also,  I believe some of the user functionality would fire multiple times. 

```
import coreStore from '@vue-storefront/store/modules/user'
import { extendStore } from 'core/lib/themes'

const getters = {
  ...
}  

export default extendStore(coreStore, {
  state,
  getters,
  mutations
})
```

**PLEASE NOTE: This issue was occurring prior to the newest store modifications.**

**Comment by pkarw** [comment]
This will be closed after we close #1102

**Comment by pkarw** [comment]
Fixed with the latest change of vuex module registration
